packages/zmarkdown/README.md

```diff
@@ -1,161 +1,243 @@
+### Request `opts` values
 
-### Request
+| Name | Type | Description |
+| - | - | - |
+| `opts.images_download_dir` | bool | [see `/latex`][ref-latex] |
+| `opts.local_url_to_local_path` | string | [see `/latex`][ref-latex] |
 
-* `opts.disable_images_download`, bool, default: `false`
+### Response `metadata` values
 
-  Do not download images.
+| Name | Type | Description |
+| - | - | - |
+| `metadata.disableToc` | bool | Whether or not the input Markdown did **not** contain headings (`#`, `##`, …). This property is named that way because we use it to disable Table of Contents generation when no headings were found.<br>- `disableToc: true` means *no headings*<br>- `disableToc: false` means at least one *heading*. |
 
-* `opts.images_download_dir`, string
+## html
 
-  Where to download the images to.
+Markdown to HTML
 
-* `opts.local_url_to_local_path`, \[from: string, to: string\], default: `<none>`
+### URL
 
-  If provided, local images referenced in Markdown source (such
-  as `![](/img/example.png)`) will be copied to `images_download_dir`
-  after replacing the string `from` with `to` using the following RegExp:
+```
+POST http://localhost:27272/html
+```
 
-  ```js
-  '/img/example.png'.replace(new RegExp(`^${from}`), to)
-  ```
+### Request `opts` values
 
-* `opts.images_download_timeout`
+| Name | Type | Description |
+| - | - | - |
+| `opts.disable_ping` | bool | default: `false`, [pings][ping] won't get parsed. |
+| `opts.disable_jsfiddle` | bool |  default: `false`, JSFiddle [iframes][iframes] are disabled. |
+| `opts.inline` | bool | default: `false`, Only parse inline Markdown elements (such as links and emphasis, unlike lists and fenced code blocks). |
+| `opts.stats` | bool | default: `false`, Will compute and return statistics about markdown text. |
 
-  HTTP request timeout for each image, in milliseconds. Defaults to 5000 ms.
+### Response `metadata` values
 
-* `opts.disable_jsfiddle`: see `/html`
+| Name | Type | Description |
+| - | - | - |
+| `metadata.disableToc` | bool | Whether or not the input Markdown did **not** contain headings (`#`, `##`, …). This property is named that way because we use it to disable Table of Contents generation when no headings were found.<br>`disableToc: true` means *no headings*<br>`disableToc: false` means at least one *heading* |
+| `metadata.ping` | string[] | undefined if `opts.disable_ping: true`<br>The list of nicknames returned by `remark-ping`. Can be used to send "ping" notifications to the corresponding users.<br>Note: this is fully customizable, `remark-ping` can validate potential *ping*s by any means, including sending an HTTP request (we recommend `HEAD`) to a REST API to make sure this username actually exists. |
+| `metadata.languages` | string[] | A list of unique languages used in GitHub Flavoured Markdown fences with a flag. |
+| `metadata.stats` | object | stats about the parsed text:<br>- `signs`: number of chars, spaces included.<br>- ` words`: number of words. |
```

Comment:
A small but complete example (request/response) would be useful for this endpoint since it's the most useful one. Ideally something with >=2 fenced code blocks featuring different languages, >= 2 pings, etc.